libmpq/mpq-internal.h

```diff
@@ -64,7 +64,14 @@
 #define TRUE 1
 #endif
 
-#include "pack_begin.h"
+#ifdef _MSC_VER
+#pragma pack(push,1)
+#define PACK_STRUCT
```

Comment:
Won't we get a macro redefinition warning since this is defined in 2 headers?
Consider adding `#undef PACK_STRUCT` at the end

Reply:
good point. fixed 🙂